src/main/interactive_shell.cpp

```diff
@@ -312,19 +321,24 @@ bool InteractiveShell::readAndExecCommands()
 
   /* There may be more than one command in the input. Build up a
      sequence. */
-  std::vector<std::unique_ptr<Command>> cmdSeq;
-  std::unique_ptr<Command> cmdp;
+  std::vector<Command> cmdSeq;
+  Command cmdp;
```

Comment:
What does `cmdp` stand for? Also, the life time of this could be shortened to one iteration of the while loop.

Reply:
Originally, "command pointer" i think. Could change this name.